preorder.html

```diff
@@ -0,0 +1,164 @@
+          <tr class="price">
+            <td>Pre-order Special</td>
+            <td>$6 <br /><small>/ month</small></td>
+            <td>$12 <br /><small>/ month</small></td>
+            <td>$24 <br /><small>/ month</small></td>
+          </tr>
+          <tr>
+            <td></td>
+            <td>
+              <form class="preorder" action="https://payments.sandstorm.io/pre-order-standard-instance" method="POST">
+                <script
+                  src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                  data-key="pk_test_euywwlSEioegnYyeZPPjYGZ2"
+                  data-name="Sandstorm"
+                  data-image="/images/sandstorm-purplecircle.png"
+                  data-description="Pre-order Standard Instance"
+                  data-amount="600"
+                  data-label="Standard >>"
+                  data-allow-remember-me="false"
+                  data-panel-label="{{amount}} / Month">
+                </script>
+              </form>
+            </td>
+            <td>
+              <form class="preorder" action="https://payments.sandstorm.io/pre-order-large-instance" method="POST">
+                <script
+                  src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                  data-key="pk_test_euywwlSEioegnYyeZPPjYGZ2"
+                  data-name="Sandstorm"
+                  data-image="/images/sandstorm-purplecircle.png"
+                  data-description="Pre-order Large Instance"
+                  data-amount="1200"
+                  data-label="Large >>"
+                  data-allow-remember-me="false"
+                  data-panel-label="{{amount}} / Month">
+                </script>
+              </form>
+            </td>
+            <td>
+              <form class="preorder" action="https://payments.sandstorm.io/pre-order-mega-instance" method="POST">
+                <script
+                  src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                  data-key="pk_test_euywwlSEioegnYyeZPPjYGZ2"
+                  data-name="Sandstorm"
+                  data-image="/images/sandstorm-purplecircle.png"
+                  data-description="Pre-order Mega Instance"
+                  data-amount="2400"
+                  data-label="Mega >>"
+                  data-allow-remember-me="false"
+                  data-panel-label="{{amount}} / Month">
+                </script>
+              </form>
+            </td>
+          </tr>
+        </table>
+        <hr />
+
+        <p id="resource-units">
+          <span id="resource-units-star">*</span>
+          A resource unit is 1 GB of RAM in continuous over use a full month. Static web content consumes 0 resource units. 200 resource units should be more than plenty for 3 typical apps being constantly used. Apps' resource utilization can vary based on database size and which features of the app are in use. <a href="#">Read More >></a>
```

Comment:
Hmm, this text is not correct. Also, we are renaming these to "compute units". So, replace with:

A compute unit is 1 GB of RAM used for one hour. Sandstorm apps only use RAM while you are actively using them, so 200 CU should be far more than enough for most users. Learn more about compute units >>